src/switch_core_media.c

```diff
@@ -1895,6 +1894,7 @@ SWITCH_DECLARE(int) switch_core_session_check_incoming_crypto(switch_core_sessio
 		if (engine->type == SWITCH_MEDIA_TYPE_AUDIO) {
 			switch_channel_set_variable(session->channel, "srtp_remote_audio_crypto_key", crypto);
 			switch_channel_set_variable_printf(session->channel, "srtp_remote_audio_crypto_type", "%s", switch_core_media_crypto_type2str(ctype));
+			switch_channel_set_variable_printf(session->channel, "rtp_last_audio_local_crypto_key", "%s", engine->ssec[engine->crypto_type].local_crypto_key);
```

Comment:
What about `VIDEO` and `TEXT`? See `"rtp_last_%s_local_crypto_key", type2str(type)`